frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -263,398 +269,105 @@ export function SurveyForm({ id }: { id: string }): JSX.Element {
     )
 }
 
-export function SurveyView({ id }: { id: string }): JSX.Element {
-    const { survey, dataTableQuery, surveyLoading, surveyPlugin, installingPlugin } = useValues(surveyLogic)
-    // TODO: survey results logic
-    // const { surveyImpressionsCount, surveyStartedCount, surveyCompletedCount } = useValues(surveyResultsLogic)
-    const { editingSurvey, updateSurvey, launchSurvey, stopSurvey, archiveSurvey, installSurveyPlugin } =
-        useActions(surveyLogic)
-    const { deleteSurvey } = useActions(surveysLogic)
+export function SurveyReleaseSummary({
+    id,
+    survey,
+    targetingFlagFilters,
+}: {
+    id: string
+    survey: Survey | NewSurvey
+    targetingFlagFilters?: Pick<FeatureFlagFilters, 'groups'> | null
+}): JSX.Element {
     const { cohortsById } = useValues(cohortsModel)
-    const { editPlugin } = useActions(pluginsLogic)
-    const [setupModalIsOpen, setSetupModalIsOpen] = useState(false)
-    const { user } = useValues(userLogic)
-
-    const [tabKey, setTabKey] = useState(survey.start_date ? 'results' : 'overview')
-    useEffect(() => {
-        if (survey.start_date) {
-            setTabKey('results')
-        }
-    }, [survey.start_date])
-
     return (
-        <div>
-            {surveyLoading ? (
-                <LemonSkeleton />
-            ) : (
+        <div className="flex flex-col mt-2 gap-2">
+            <div>
+                {survey.linked_flag_id || survey.conditions?.url || survey.conditions?.selector || targetingFlagFilters
+                    ? 'This survey will be released to users who match all of the following:'
+                    : 'This survey will be released to everyone'}
+            </div>
+            {survey.linked_flag_id && (
+                <div className="flex flex-row font-medium gap-1">
+                    <span>Are part of the feature flag:</span>{' '}
```

Comment:
I'm guessing these are all AND conditions but the UI isn't super clear on that. And it would be _really_ helpful to see here what the feature flag conditions are so I don't have to click back and forth between the flag and the survey to know who will be included.

Reply:
I have this line here but is that not super clear? Would it be clearer if I added `AND` to each of the conditions?

<img width="393" alt="Screen Shot 2023-06-20 at 1 14 30 PM" src="https://github.com/PostHog/posthog/assets/25164963/5f2f28d7-8a8f-4510-bacf-53200ddb95b8">

I can also add the feature flag conditions in a follow up PR since that's a bit more complex in terms of UI display